Jenkinsfile

```diff
@@ -283,11 +283,6 @@ def e2eTest() {
           e2eGKE('tight', 'http', 'managed')
         }
       }],
-      ['gke-loose-http', {
```

Comment:
What is the difference between `loose` vs `tight` vs `custom`? It is hard for me to review `e2e-kube.sh` without understanding this.

Reply:
"tight" is side-car.   "loose" is not side-car, proxy is installed at a separate pod,  since we don't support it anyway,  so I removed it.
"custom" is custom nginx config,